### ISSUE TITLE
oxlint.config.mts file not detected

### ISSUE THREAD

**Issue by bbrzoska** [oxc-project/oxc-vscode#174]
### Confirm that this is a Bug for the editor extension, not the CLI

- [x] This bug does not happen in the Oxlint/Oxfmt CLI
- [x] After restarting the editor, the bug still appears

### Versions used

```
VS Code Extension v1.50.0
oxlint is enabled (v1.56.0)

Operating System and Version:
Node Version: v24.13.1
Node Manager: prototools
```

### What happened?

oxlint.config.mts is not auto-detected by oxc-vscode. The config gets picked up just fine in CLI though. Workaround is to set `"oxc.configPath": "./oxlint.config.mts"` manually.

Same issue in Zed: https://github.com/oxc-project/oxc-zed/issues/149

### Reproduction repo

1. Create "./oxlint.config.mts" in repo.
2. Open a file with linting errors that get picked up by CLI - they don't get picked up by the VSCode plugin

### Output channel log

<details>
<summary>Click to expand</summary>
2026-03-24 14:00:08.493 [info] 2026-03-24T21:00:08.486064Z  INFO oxc_language_server::backend: initialize: Some([WorkspaceOption { workspace_uri: Uri(Uri { scheme: "file", authority: Some(Authority { userinfo: None, host: "", host_parsed: RegName(""), port: None }), path: "~/myproject", query: None, fragment: None }), options: Object {"disableNestedConfig": Bool(false), "fixKind": String("safe_fix"), "run": String("onType"), "flags": Object {"disable_nested_config": String("false"), "fix_kind": String("safe_fix")}} }])
2026-03-24T21:00:08.486252Z  INFO oxc_language_server::backend: oxlint version: 1.56.0
</details>
